drv/lpc55-update-api/src/lib.rs

```diff
@@ -242,6 +242,16 @@ impl From<RotSlot> for SlotId {
     }
 }
 
+impl SlotId {
+    pub fn other(&self) -> SlotId {
+        if *self == SlotId::A {
+            SlotId::B
+        } else {
+            SlotId::A
+        }
```

Comment:
```suggestion
        match self {
            SlotId::A => SlotId::B,
            SlotId::B => SlotId::A,
        }
```
slightly shorter, and will break loudly if someone adds a third slot